.gitignore

```diff
@@ -2,5 +2,4 @@ node_modules
 .vscode/*
 .env
 .code-workspace
-*.env
```

Comment:
we are adding a template env file called `sample.env`. setup involves creating a new file called .env via `cp sample.env .env` this way folks can have their envars correct without the risk of secrets being pushed.